front/app/containers/Admin/routes.tsx

```diff
@@ -71,6 +71,9 @@ const getRedirectURL = (
   }
 
   // if not, redirect them to the sign-in page
+  // TO DO: Here we need to redirect to index
+  // if the user is already signed in. If I want to access the admin as a signed in user
+  // I get redirect => sign in => then index (because I'm already signed in)
```

Comment:
Only relevant if we rely on the redirection from the routes.tsx file I mention above

Reply:
And it still works, just a bump in between to /sign-in